app/src/main/java/com/mycompany/myapp/data/DataModule.java

```diff
@@ -29,4 +39,46 @@ public OkHttpClient provideOkHttpClient(Cache cache) {
                 .cache(cache)
                 .build();
     }
+
+    @Singleton
+    @Api("github")
+    @Provides
+    public String provideBaseUrl(Settings settings) {
+        return settings.getBaseUrl();
+    }
+
+    @Singleton
+    @Api("github")
+    @Provides
+    public Converter.Factory provideConverter() {
+        Gson gson = new Gson();
+        return GsonConverterFactory.create(gson);
+    }
+
+    @Singleton
+    @Api("github")
+    @Provides
+    public Retrofit provideRetrofit(
+            OkHttpClient client,
+            @Api("github") String baseUrl,
+            @Api("github") Converter.Factory converterFactory) {
+        return new Retrofit.Builder()
+                .client(client)
+                .baseUrl(baseUrl)
+                .addConverterFactory(converterFactory)
+                .addCallAdapterFactory(RxJavaCallAdapterFactory.create())
+                .build();
+    }
+
+    @Singleton
+    @Provides
+    public GitHubApiService provideGitHubApiService(@Api("github") Retrofit retrofit) {
```

Comment:
Just to answer where it's coming from, it's defined under `com.getfinal.android.data.api` in main.  It's basically just saying this Retrofit instance is for github.  For instance, there could be another Retrofit instance with `@Api("bitbucket")` and that instance would be for bitbucket.